backend/src/modules/auth/auth.service.ts

```diff
@@ -22,7 +22,7 @@ export class AuthService {
     throw new UnauthorizedException('User or password are incorrect');
   }
 
-  async login(user: LoginUserEntity) {
+  async login(user: LoginUserEntity): Promise<{ id: string; email: string; token: string }> {
```

Comment:
Create new folder with types and create new file auth.types.ts and move this return type to this file